lib/core/src/system/gpu/data/uniform.rs

```diff
@@ -207,6 +207,13 @@ impl<Value> {
     pub fn unset_dirty(&mut self) {
         self.dirty = false;
     }
+}
+
+impl<Value:Clone> {
+    /// Reads the value of this uniform.
+    pub fn get(&self) -> Value {
+        self.value.clone()
+    }
```

Comment:
When someone implements Clone, the cloning may be an expensive operation. When I see method "get" in Uniform, I don't expect a possible heavy operation.

Reply:
Ok, after talking via phone I agreed, but now I dont see a clear way how to do it. Lets talk again.